src/domUpdates.js

```diff
@@ -53,6 +55,34 @@ const showSearchResults = (userValue, names, images, ids) => {
   }
 }
 
+const displayAllRecipes = (recipeData) => {
+  hideAllPages();
+  allRecipesView.classList.remove("hidden")
+  allRecipesSection.innerHTML = ''
+  const recipeDataAlpha = recipeData.sort((a, b) => {
+    const nameA = a.name.toUpperCase();
```

Comment:
Great job figuring out the sorting alphabetically! 
Let's break this into its own function so that we can reuse it. It should go into the file where we manipulate the data.